support-frontend/assets/components/checkoutBenefits/newBenefitPill.tsx

```diff
@@ -0,0 +1,19 @@
+import { css } from '@emotion/react';
+import {
+	neutral,
+	news,
+	space,
+	textSansBold14,
+} from '@guardian/source/foundations';
+
+const newBenefitPill = css`
+	background-color: ${news[400]};
+	color: ${neutral[100]};
+	${textSansBold14};
+	border-radius: 4px;
+	padding: 0 ${space[1]}px;
+`;
+
+export function NewBenefitPill() {
+	return <span css={newBenefitPill}>New</span>;
+}
```

Comment:
If we don't use this anywhere outside of the benefits, should we keep it in that component?

Reply:
I think it's part of the design of the product banner
![image](https://github.com/user-attachments/assets/1b2095ed-d0c7-4c7c-9a9f-44eab78381b0)